### DISCUSSION TITLE
Why I can't access my private repository？

### DISCUSSION THREAD

**Discussion by liao20081228** [community/community#23028] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Everyone!</p>
<p>Today I found <a href="http://github.com" rel="nofollow noopener">github.com</a> can create a private repository for free. So  <strong>I changed my public repository to a private repository via “setting-&gt;option-&gt; make private”, Unfortunately，I can’t access it after that.</strong> I can see it on my repository management homepage, but when I click on the link, <strong>the web browser prompts: “oops 500”</strong>.I found everything is  very good on <a href="http://gitstatus.com" rel="nofollow noopener">gitstatus.com</a> . What happened to him. What should I do to access it? <br>
                      Thanks！</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Unfortunately, we don’t have access to private account details here in the Community Forum. In order to find out what might be going wrong with your repository, you’ll have to contact private support at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> or by email at <a href="mailto:support@github.com">support@github.com</a>.</p>
<p>I hope that helps!</p>

**Comment by liao20081228** [comment]
<p>Thank you for the replay, YES, I connected the support of github ,and after one week, this probelm had be solved.</p>
